crates/starlight/src/vm/value.rs

```diff
@@ -591,7 +591,7 @@ impl JsValue {
         if unlikely(number.is_nan() || number.is_infinite()) {
             return Ok(0);
         }
-        Ok(number.floor() as i32)
```

Comment:
what's the difference between current and your impl there?

Reply:
number.floor() will be 0 if number <0